locale/ko/download/package-manager.md

```diff
@@ -0,0 +1,788 @@
+---
+layout: page.hbs
+title: 패키지 매니저로 Node.js 설치하기
+---
+
+<!--
+# Installing Node.js via package manager
+
+***Note:*** The packages on this page are maintained and supported by their respective packagers, **not** the Node.js core team. Please report any issues you encounter to the package maintainer. If it turns out your issue is a bug in Node.js itself, the maintainer will report the issue upstream.
+
+----------------------------
+
+* [Arch Linux](#arch-linux)
+* [Debian and Ubuntu based Linux distributions](#debian-and-ubuntu-based-linux-distributions)
+* [Enterprise Linux and Fedora](#enterprise-linux-and-fedora)
+* [FreeBSD and OpenBSD](#freebsd-and-openbsd)
+* [Gentoo](#gentoo)
+* [NetBSD](#netbsd)
+* [openSUSE and SLE](#opensuse-and-sle)
+* [OSX](#osx)
+* [SmartOS and illumos](#smartos-and-illumos)
+* [Void Linux](#void-linux)
+* [Windows](#windows)
+
+----------------------------
+-->
+# 패키지 매니저로 Node.js 설치하기
+
+***Note:*** 이 페이지에 나오는 패키지는 각 패키지 관리자가 관리하고 Node.js 코어 팀이
+**관리하지 않습니다**. 이슈가 있다면 패키지 관리자에게 보고해 주세요. 해당 이슈가 Node.js 자체의
+버그라면 관리자가 이슈를 Node.js에 보고할 것입니다.
+
+----------------------------
+
+* [Arch Linux](#arch-linux)
+* [Debian과 Ubuntu에 기반을 둔 Linux 배포판](#debian-and-ubuntu-based-linux-distributions)
```

Comment:
Oh, I missed this one. 
This link will not work since The anchors based on h2 tag contents.